fission-web-server/README.md

```diff
@@ -41,3 +41,49 @@ $ psql
 ```
 
 Migrations will be performed automatically when running the server
+
+### Using Docker
+
+To mimic the full "fission stack" for local development, you can use the included `docker-compose.yml`. This assumes that you have Docker installed locally:
+
+1. Copy the env.example file to `.env` (these values are used by docker compose). Edit as desired (defaults will work)
```

Comment:
Since we also have `env.yaml`, that may get confusing. Any thoughts on changing that to `env.yaml -> server.yaml` _or_ `.env -> .docker`?

Reply:
I'm pro the idea `env.yaml -> server.yaml`. `.env` seems to be somewhat of a canonical name for that to me.

Reply:
I agree that the `.env` file name is pretty canonical (bonus `direnv` will pick it up if you're doing shell things with those values).

So I'm clear - are you proposing `server.yaml.example` or changing the server code to load `server.yaml` rather than `env.yaml`?

Reply:
> So I'm clear - are you proposing server.yaml.example or changing the server code to load server.yaml rather than env.yaml?

Both - (at least that's what I'd be proposing)

Reply:
Sounds good to me! 👍